packages/compass-components/src/components/signal-popover.tsx

```diff
@@ -110,6 +124,7 @@ const SignalCard: React.FunctionComponent<
   primaryActionButtonLink,
   darkMode: _darkMode,
   onPrimaryActionButtonClick,
+  cardTitleStyles,
```

Comment:
Nit: this is a component internal to this whole file, we probably don't need to be so generic with this prop, might be easier to just pass multiSignals / isMultiSignal here and use this to apply the className

Reply:
Sounds good, I just added this 👍